assets/user_data.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -ex
 
-exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
+# exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
```

Comment:
Why did you comment this out? we need to have the user-data.log file to review any failed launches

Reply:
Sorry, I didn't mean to commit this. I'll revert it.

It was a change I had made while trying to understand a EBS attachment failure.

EC2 user data script output is normally logged to /var/log/cloud-init-output.log.

I was surprised that the EBS error message was not being logged there.

It turns out that this line was redirecting the output to /var/log/user-data.log.

If that's where you expect to find the logs, then there is no need for me to change it.